forge/forge.js

```diff
@@ -85,12 +85,13 @@ module.exports = async (options = {}) => {
     }
 
     if (runtimeConfig.telemetry.backend?.sentry?.dsn) {
+        const environment = process.env.SENTRY_ENV ?? (process.env.NODE_ENV ?? 'unknown')
         server.register(require('@immobiliarelabs/fastify-sentry'), {
             dsn: runtimeConfig.telemetry.backend.sentry.dsn,
-            environment: process.env.SENTRY_ENV ?? (process.env.NODE_ENV ?? 'unknown'),
+            environment,
             release: `flowfuse@${runtimeConfig.version}`,
-            tracesSampleRate: 0.1,
```

Comment:
FWIW it doesn't seem like this config option is being respected, am looking into _why_.